python/jittor/init.py

```diff
@@ -706,26 +706,26 @@ def norm_cdf(x):
                       "The distribution of values may be incorrect.",
                       stacklevel=2)
 
-    with jt.no_grad():
-        # Values are generated by using a truncated uniform distribution and
-        # then using the inverse CDF for the normal distribution.
-        # Get upper and lower cdf values
-        l = norm_cdf((a - mean) / std)
-        u = norm_cdf((b - mean) / std)
-
-        # Uniformly fill tensor with values from [l, u], then translate to
-        # [2l-1, 2u-1].
-        # var.uniform(2 * l - 1, 2 * u - 1)
-        jt.init.uniform_(var, low=2 * l - 1, high=2 * u - 1)
-
-        # Use inverse cdf transform for normal distribution to get truncated
-        # standard normal
-        var.erfinv()
-
-        # Transform to proper mean, std
-        var.multiply(std * math.sqrt(2.))
-        var.add(mean)
-
-        # Clamp to ensure it's in the proper range
-        var.clamp(min_v=a, max_v=b)
-        return var
\ No newline at end of file
+
+    # Values are generated by using a truncated uniform distribution and
+    # then using the inverse CDF for the normal distribution.
+    # Get upper and lower cdf values
+    l = norm_cdf((a - mean) / std)
+    u = norm_cdf((b - mean) / std)
+
+    # Uniformly fill tensor with values from [l, u], then translate to
+    # [2l-1, 2u-1].
+    # var.uniform(2 * l - 1, 2 * u - 1)
+    var.uniform_(low=2 * l - 1, high=2 * u - 1)
+
+    # Use inverse cdf transform for normal distribution to get truncated
+    # standard normal
+    var = var.erfinv()
+
+    # Transform to proper mean, std
+    var = var.multiply(std * math.sqrt(2.))
+    var.add(mean)
```

Comment:
add, clamp are not inplace function too, inplace function will end with underscore

Reply:
Ok, it is done!